packages/dht/src/connection/WebSocket/WebSocketConnector.ts

```diff
@@ -137,6 +145,7 @@ export class WebSocketConnector implements IWebSocketConnectorService {
             }
         } catch (err) {
             if (reattempt < 5) {
```

Comment:
Maybe a separate PR:
- could introduce constants for these magic numbers.
- maybe implement the retry logic with retry helper like in `ProxyStreamConnectionClient#24`, or with simple for loop?